indexer/src/configs.rs

```diff
@@ -72,6 +74,13 @@ impl Opts {
             ChainId::Testnet(_) => "https://rpc.testnet.near.org",
         }
     }
+
+    pub fn genesis_file_url(&self) -> &str {
+        match self.chain_id {
+            ChainId::Mainnet(_) => "https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore-deploy/mainnet/genesis.json",
+            ChainId::Testnet(_) => "https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore-deploy/testnet/genesis.json",
```

Comment:
I think we need somehow handle the case that @mhalambek is handling in #339 by providing the `Custom` chain id. But I am not sure how would be the best way, though.

Reply:
Perhaps we might handle that separately after PR #339 is finished.

Reply:
Let's wait til either of these PR's are merged, then we can address compatibility between the them in the one which isn't merged :)